src/Tizen.MachineLearning.Train/Tizen.MachineLearning.Train/Model.cs

```diff
@@ -0,0 +1,409 @@
+        /// </remarks>
+        /// <param name="filePath">File path to save the file.</param>
+        /// <param name="format">Format flag to determine which format should be used to save.</param>
+        /// <since_tizen> 10 </since_tizen>
+        public void Save(string filePath, NNTrainerModelFormat format)
+        {
+            if (string.IsNullOrEmpty(filePath))
+                NNTrainer.CheckException(NNTrainerError.InvalidParameter, "File path is null");
+            Log.Info(NNTrainer.Tag, "File path: "+ filePath);
+            NNTrainerError ret = Interop.Model.Save(handle, filePath, format);
+            NNTrainer.CheckException(ret, "Failed to save model to path");
+        }
+
+        /// <summary>
+        /// Loads the model.
+        /// </summary>
+        /// <remarks>
+        /// Use this function to load the current model. format
+        /// describes various formats in which various selections of the
+        /// parameters of the models can be loaded. Some formats may load
+        /// parameters required for training. Some other formats may load model
+        /// configurations. Unless stated otherwise, loading model configuration requires
+        /// a freshly constructed model with new model() without Compile(),
+        /// loading model parameter requires Compile() to be called upon the model
+        /// before calling this function.
+        /// If you want to access only internal storage by using this function,
+        /// you should add privilege %http://tizen.org/privilege/mediastorage. Or, if you
+        /// want to access only external storage by using this function, you should add
+        /// privilege %http://tizen.org/privilege/externalstorage. If you want to access
+        /// both storage, you must add all the privileges.
+        /// </remarks>
+        /// <param name="filePath">File path to load the file.</param>
+        /// <param name="format">Format flag to determine which format should be used to load.</param>
+        /// <since_tizen> 10 </since_tizen>
+        public void Load(string filePath, NNTrainerModelFormat format)
+        {
+            if (string.IsNullOrEmpty(filePath))
+                NNTrainer.CheckException(NNTrainerError.InvalidParameter, "File path is null");
+            Log.Info(NNTrainer.Tag, "File Path: "+ filePath);
+            NNTrainerError ret = Interop.Model.Load(handle, filePath, format);
+            NNTrainer.CheckException(ret, "Failed to load model to path");
+        }
+
+        /// <summary>
+        /// Adds layer in neural network model.
+        /// </summary>
+        /// <remarks>
+        /// Use this function to add a layer to the model. The layer is added to
+        /// the end of the existing layers in the model. This transfers the
+        /// ownership of the layer to the network. No need to destroy the layer once it
+        /// is added to a model. layer is available until the model is released. so
```

Comment:
The https://github.com/Samsung/TizenFX/pull/4156/files#r920596188 looks better, or at least change:
```suggestion
        /// is added to a model. Layer is available until the model is released, so
```

Reply:
thank you